pom.xml

```diff
@@ -48,7 +48,7 @@
     <checkstyleFormatter>${project.basedir}/contrib/checkstyle.xml</checkstyleFormatter>
     <dep.commons-codec.version>1.15</dep.commons-codec.version>
     <dep.commons-collections.version>4.4</dep.commons-collections.version>
-    <dep.commons-compress.version>1.21</dep.commons-compress.version>
+    <dep.commons-compress.version>1.24.0</dep.commons-compress.version>
```

Comment:
Is it worth going to 1.25.0?

Reply:
Is that available?

Reply:
I don't know. I forgot to check. We can just go with 24 for now